numpyro/contrib/nested_sampling.py

```diff
@@ -260,7 +260,6 @@ def prior_model() -> PriorModelGen:
             num_live_points=model.U_ndims * 25,
             num_parallel_samplers=1,
             max_samples=1e4,
-            uncert_improvement_patience=2,
```

Comment:
intentional?

Reply:
yup, this is causing CI failing in master branch.